src/routes/dashboard/send/+page.svelte

```diff
@@ -175,19 +177,27 @@ features have been implemented:
      * @function previewPaymentTransaction
      */
     const previewPaymentTransaction = async () => {
+        const destinationAddress = otherDestination ? otherPublicKey : destination;
         let { transaction, network_passphrase } = createAccount
             ? await createCreateAccountTransaction({
                   source: data.publicKey,
-                  destination: otherDestination ? otherPublicKey : destination,
+                  destination: destinationAddress,
                   amount: sendAmount,
                   memo: memo,
               })
+            : StrKey.isValidContract(destinationAddress)
```

Comment:
Let's use the helpful contract check you wrote into the `contactsStore` here. Then, you won't even need to import `StrKey` from the stellar sdk
```suggestion
            : contacts.isContractAddress(destinationAddress)
```

Reply:
Done